.github/workflows/deploy.yml

```diff
@@ -15,4 +15,4 @@ jobs:
           node-version: 12.x
       - run: echo $GITHUB_SHA > static/sha
       - run: now --version
-      - run: now -S badgen alias $(now -S badgen --token ${{ secrets.ZEIT_TOKEN }} -c) flat.badgen.net --token ${{ secrets.ZEIT_TOKEN }}
+      - run: now -S badgen alias $(now -S badgen --token ${{ secrets.VERCEL_TOKEN }} -c) flat.badgen.net --token ${{ secrets.VERCEL_TOKEN }}
```

Comment:
You should use `vercel` instead 🙂

Reply:
I added this job to make flat.badgen.net a standalone deployment, other than sharing with badgen.net, since Vercel cache works at deployment level, not url level.